client/src/Component/homepage/homepage.js

```diff
@@ -22,7 +28,27 @@ useEffect(()=>{
 
   return (
     <div className={dark ? "landing-page-orange dark-mode":"landing-page-orange"}>
-       
+    <div className="landing-page-orange">
+      <div className="social">
+        <ul>
+          <li><a><span>WhatsApp</span><WhatsAppIcon /></a></li>
+          <li><a><span>Facebook</span><FacebookIcon /></a></li>
+          <li><a><span>Github</span><GitHubIcon /></a></li>
+          <li><a><span>Instagram</span><InstagramIcon /></a></li>
+          <li><a><span>YouTube</span><YouTubeIcon /></a></li>
+          <li><a><span>LinkedIn</span><LinkedInIcon /></a></li>
+          </ul>
+        </div>
+
+        <div className="header">
+            <img className="vector-3" src={logo} alt="vector-3" />
+            <div className="home">
+              <img className="vector-4" src={vector5} alt="vector-4" />
+              <div className="place montserrat-bold-white-24px">{place}</div>
+            </div>
+            <div className="Auth">
+            <Link to="/signin"><div className="sign-in montserrat-bold-white-24px">{signIn}</div></Link>
+            <Link to="/signup"><div className="sign-up montserrat-bold-white-24px">{signUp}</div></Link>
```

Comment:
You need to add  /div and remove error from your code.

Reply:
![temp](https://user-images.githubusercontent.com/61119120/115180966-74faf400-a0f4-11eb-818a-bfa38481b624.png)